BraintreeCore/src/main/java/com/braintreepayments/api/core/GetPaypalLaunchTypeUseCase.kt

```diff
@@ -0,0 +1,38 @@
+package com.braintreepayments.api.core
+
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class GetPaypalLaunchTypeUseCase(
```

Comment:
Can we add a Kdoc for this class - it'll help give context on when and why it's used.